codacy/values.yaml

```diff
@@ -369,6 +369,21 @@ minio:
     size: 20Gi
   minioConfig:
     region: "eu-west-1"
+  buckets:
+  - name: "analysis"
+    policy: "public"
```

Comment:
as per the chart's documentation, the available policies are none|download|upload|public.